docs/en/samples/arm64.md

```diff
@@ -3,7 +3,7 @@
 ## Prerequisites
 
 ### Get the latest code of Fluid
-
+We currently provide the official image of Fluid for multiple platforms. You can refer to the [document](../userguide/get_started.md) to deploy Fluid diretly and jump to the step of *Example*.
```

Comment:
The official image for  multiple platforms is provided by Fluid community.

Reply:
fixed in https://github.com/fluid-cloudnative/fluid/pull/2154/commits/cb6304ed170a5f5254bd193223c7d4708e8d4d9f